test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -778,10 +779,18 @@ protected Settings featureFlagSettings() {
         for (Setting builtInFlag : FeatureFlagSettings.BUILT_IN_FEATURE_FLAGS) {
             featureSettings.put(builtInFlag.getKey(), builtInFlag.getDefaultRaw(Settings.EMPTY));
         }
```

Comment:
This is the right way to add `FeatureFlags.TELEMETRY_SETTING`, why it was removed?

Reply:
Added it back.